From 4a83e35d2f8050f9db80e28bb22490fcd7ae1663 Mon Sep 17 00:00:00 2001
From: Nikolay Aleksandrov <nikolay@redhat.com>
Date: Mon, 1 Sep 2014 13:33:03 -0400
Subject: [crypto] qat: Fix build problem with O=

Message-id: <1409578401-26943-12-git-send-email-nikolay@redhat.com>
Patchwork-id: 90088
O-Subject: [RHEL7.1 PATCH BZ 1091130 11/29] crypto: qat - Fix build problem with O=
Bugzilla: 1091130
RH-Acked-by: Prarit Bhargava <prarit@redhat.com>
RH-Acked-by: Herbert Xu <herbert.xu@redhat.com>

From: Herbert Xu <herbert@gondor.apana.org.au>

qat adds -I to the ccflags.  Unfortunately it uses CURDIR which
breaks when make is invoked with O=.  This patch replaces CURDIR
with $(src) which should work with/without O=.

Signed-off-by: Herbert Xu <herbert@gondor.apana.org.au>
(cherry picked from commit e60b244281cfe03ddd7c5c15c4e6b6d6316bb530)
Signed-off-by: Jarod Wilson <jarod@redhat.com>

diff --git a/drivers/crypto/qat/qat_dh895xcc/Makefile b/drivers/crypto/qat/qat_dh895xcc/Makefile
index 8e4924d..25171c5 100644
--- a/drivers/crypto/qat/qat_dh895xcc/Makefile
+++ b/drivers/crypto/qat/qat_dh895xcc/Makefile
@@ -1,4 +1,4 @@
-ccflags-y := -I$(CURDIR)/drivers/crypto/qat/qat_common
+ccflags-y := -I$(src)/../qat_common
 obj-$(CONFIG_CRYPTO_DEV_QAT_DH895xCC) += qat_dh895xcc.o
 qat_dh895xcc-objs := adf_drv.o \
   adf_isr.o \
-- 
1.7.1